jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -211,11 +211,15 @@ def get_regions(self, region_type=None, list_of_subset_labels=None,
             the native data unit. If True, subsets are returned in the spectral
             axis display unit set in the Unit Conversion plugin.
 
-        return_sky_region : bool or None, optional
-            If None (default) or True, then the returned region will be ``SkyRegion`` if the
-            configuration is Imviz and the data is aligned by WCS, or if the configuration
-            is Cubeviz and the data has a WCS'. If set to False, a ``PixelRegion`` object will
-            be returned.
+        wrt_data : bool, str,  or None
+            Only applicable for spatial subsets, an error will be raised for spectral subsets.
+            If aligned by WCS and ``wrt_data`` is not False, return ``SkyRegion`` object. In this
+            case, if ``wrt_data`` is set to the name of data, return ``SkyRegion`` using the wcs of
+            that data. Otherwise, the default is to use the parent data of the subset. If the
+            parent of the subset is the Default orientation layer, the first data loaded into
+            ``data_collection`` will be used instead. Also, if aligned by pixels and ``wrt_data``
+            is True or set to the name of valid data, return ``SkyRegion``. Finally, if in the
```

Comment:
I would put the statement 'Finally, if in the cubeviz configuration, return ``SkyRegion``' in the top of the docstring since this doesn't pertain to this argument, this is just general default behavior.